### PR TITLE
Enable building teleport with the new UI location

### PR DESCRIPTION
**Pull request by hatched**
Changes necessary to have the UI build correctly with the new file locations.

### CODE REVIEW

File: .github/workflows/doc-tests.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run tests
-        run: cd /src/content && yarn markdown-lint
+        run: cd /src && yarn markdown-lint
```
Comment (zmb3) [inline .github/workflows/doc-tests.yaml:29]: This recurses into the content directory correctly?

File: .drone.yml [hunk 2]
```diff
@@ -18273,6 +18196,6 @@ volumes:
   temp: {}
 ---
 kind: signature
-hmac: df00b73f447e6ab65125e08794a80e6ec48bdbfbd6dafdc0449ab2f87c4ecdee
+hmac: f7f0c96cbbabe4305e224c72cd5f9f322b6a54325a890f1537500a5f3eb0e303
```
Comment (zmb3) [inline .drone.yml:18199]: Reminder to double check this before merging.

**Review by zmb3 — APPROVED** [review]
Looks good.

I assume the e submodule reference needs to be updated after the corresponding e PR is merged?

**Review by avatus — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (hatched) [reply .drone.yml:18199]: Yep it looks good 👍

**Pull request merged**